src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -28,48 +30,36 @@ Session replay allows you to play back user interactions on your web app and map
 * <DoNotTranslate>**Improving the user experience**</DoNotTranslate>: Discover areas of your web app that are causing users frustration. For example, you might see that users are getting lost in your navigation or clicking on buttons that don't do anything.
 * <DoNotTranslate>**Identifying issues in key user journeys**</DoNotTranslate>: Identify areas of your web app that are causing performance problems. For example, if session replay reveals the `/cart/checkout` page is experiencing excessive page load time, you'd prioritize fixing this critical user journey.
 
-Here's an example of how a session replay looks:
+Here's how a sample session replay looks:
 
 <img
     title="Session replay details"
     alt="Screenshot of session replay details"
     src={browserSessionReplayDetails}
 />
 
-    <figcaption>
-  <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Session replay**</DoNotTranslate>: Visualize your end-user's experience and correlate issues in their journey with session traces.
-</figcaption>
-
 ## How session replay works [#how-it-works]
 
-Session replay works by taking a snapshot of the end-user's DOM (Document Object Model) when the web page loads. The DOM is a hierarchical representation of all the elements on a web page, such as the text, images, and buttons. The browser agent uses a library to record DOM changes as the user interacts with the page. This data is then stored in New Relic's database and can be replayed up to 8 days after the session occurred. The replay is pixel perfect, and you can see exactly what the user's screen looks like when an error occurs.
-
-## Impact of session replay [#session-replay-impact]
+When a user loads your web page, session replay takes a snapshot of the Document Object Model (DOM), which represents the page's structure and visual elements. To ensure accurate styling information, the browser agent accesses your CSS files. The agent then integrates these styles directly into the replay recording, eliminating the need to reference external CSS files during playback. As the user interacts with the page, a library within the browser agent captures any changes made to the DOM. This captured data is then stored securely in New Relic's database. 
 
-### User data and security concerns [#data-security]
+Replays store up to:
+* 4 hours of a user session
+* 30 minutes of inactivity
 
-We take data privacy seriously, so we built session replay with:
+Data retention depends on your account. By default, browser event data is stored 8 days, but DataPlus customers have 98 days, and others may have even longer retentions.
```

Comment:
This "others may have even longer retentions" feels pretty odd and nebulous. Why mention it if we don't tell people how to get longer retention if they want it?